osu.Game.Tests/Visual/Online/TestSceneMessageNotifier.cs

```diff
@@ -28,6 +29,14 @@ public class TestSceneMessageNotifier : OsuManualInputManagerTestScene
         [SetUp]
         public void Setup()
         {
+            // We blindly mark every request as success so that ChannelManager doesn't remove our channel again.
+            if (API is DummyAPIAccess daa)
+            {
+                daa.HandleRequest = (request) => {
+                    return true;
+                };
```

Comment:
It's usually best to only handle the request that the test scene expects to be fired, not handle every single API request. See other usages of `HandleRequest`.